05/app.js

```diff
@@ -0,0 +1,61 @@
+﻿function Student(firstName, lastName) {
+	this.firstName = firstName;
+	this.lastName = lastName;
+	this.grades = {};
+}
+
+Student.prototype.addGrade = function (sub, grade) {
+	if (typeof this.grades[sub] === 'undefined') {
+		this.grades[sub] = [];
+	}
+
+	if (grade >= 1 && grade <= 6) {
+		this.grades[sub].push(grade);
+	} else {
+		console.log('Ocena musi zawierać się w przedziale od 1 do 6!');
+	}
```

Comment:
Lepiej byłoby zrobić osobną metodę, która sprawdza dane i jeśli jest ok to wykonauje je dodanie tj.
```
const grade = prmpt('Podaj ocenę'); // obieramy dane od uzytknika
if(student.isCorrectGrade(grade)) { // sprawdzamy
    studen.addGrade(subject, grade); // dopiero dodajemy
}
```
O tyle jest to ważne, że każda funkcja powinna robić jedną rzecz i wskazywać ją w nazwie.